com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -36,9 +36,11 @@ public function upgrade_1002(){
     return TRUE;
   }
```

Comment:
You're 100% sure about that? I mean, I'm ok about removing 1002, since the same thing is ran during the installation, but what about 1001? How can we know that all civihr installs have already been upgraded?